hack/validate/vendor

```diff
@@ -45,11 +45,11 @@ else
 		echo
 		echo 'The following files changed during re-vendor:'
 		echo
-		git diff --name-status HEAD -- "${vendor_files[@]}"
+		git --no-pager diff --name-status HEAD -- "${vendor_files[@]}"
```

Comment:
Curious; was this always an issue, or did something change in either defaults, in GitHub actions, or because a TTY was attached (or not)?

Wondering if there's other `git` commands we run that may need the same treatment, and in that case, if we need to set a global option for this, e.g. setting `GIT_PAGER` in either GitHub actions or for the container; https://git-scm.com/docs/git/2.26.0#Documentation/git.txt-codeGITPAGERcode

Reply:
I didn't see this happening before, so it looks like some GHA defaults changed.